### ISSUE TITLE
405 Error on API Request: components/{id}/checkout

### ISSUE THREAD

**Issue by cduerselenBX** [grokability/snipe-it#10000]
**Describe the bug**
When trying to checkout components to an asset via the API the following error occured
```json
{
  "status": "error",
  "messages": "Method not allowed",
  "payload": null
}
```

The request to access the API Endpoint looked like the following:
```http
POST {{host}}/components/1/checkout 
Accept: application/json
Content-Type: application/json
Authorization: Bearer {{bearer}}

{
    "assigned_to": 1925,
    "assigned_qty": 1

}
``` 
I checked the same setup with the sandbox environment/demo version online and it all worked without a problem. 
Anyhow I realized that the online server uses https while our dev environment has no certificate assigend to it - could this be the reason for the 405 Error (anyhow the GET Requests all work fine on our dev environment). 

Thank you very much for your support!

**To Reproduce**
1. Setup of snipe-it environment without SSL-Certificate
2. Run a Components-Checkout API Request like above

**Expected behavior**
A successful checkout of the component to an asset

**Screenshots**

**Server (please complete the following information):**
 - Web Server: Microsoft-IIS/10.0

**Comment by cduerselenBX** [comment]
Just tried it on a local docker environment and it worked finde (without SSL-Certificate) - could it be the case that the Microsoft IIS is the problem?

Thank you for your help!

**Comment by cduerselenBX** [comment]
We found the error: Our operations team installed snipe it not via git but via the ZIP-Download of the release (https://github.com/snipe/snipe-it/releases/tag/v5.1.8) - it looks like the new endpoint was not yet included in this release.

Is there a reason for this?

**Comment by snipe** [comment]
> Is there a reason for this?

Because it wasn't built when 5.1.8 was released, it was added pre-5.2.0
